addons/mail/static/src/js/composer.js

```diff
@@ -479,16 +479,21 @@ var BasicComposer = Widget.extend(chat_mixin, {
     },
 
     send_message: function () {
+        var button = this.$(".o_composer_button_send");
```

Comment:
jquery objects are conventionally named $xxxx
so, in this case $button

Reply:
Agree, also use single quote for selectors.
```
var $button = this.$('.o_composer_button_send');
```

Reply:
When should I use single quote and where should I use double ones ?

single => "internal" , technical string ?
double => messages showed on the UI ?

Reply:
Rule of thumb is to always use single quotes, except in the following cases:
- translated string (e.g. `var sendText = _t("Send")`)
- inside selectors with quotes (e.g. var $message = $('.o_message[data-document-model="partner"]'`)))